### PR TITLE
fix(button): Update buttons on the showcase.

### PR DESCRIPTION
**Pull request by creador-dev**

### CODE REVIEW

File: page-builder.html [hunk 1]
```diff
@@ -95,9 +95,9 @@ <h2 class="sui-box-title">Default Builder</h2>
 
 							</div>
 
-							<button class="sui-button sui-button-dashed">
-								<span class="sui-icon-plus" aria-hidden="true"></span>
-								Insert Button
+							<button class="sui-button sui-button--ghost sui-button--dashed sui-button--height-60 sui-button--full-width">
+								<span class="sui-button__icon sui-icon-plus" aria-hidden="true"></span>
```
Comment (iamleigh) [inline page-builder.html:99]: @creador-dev, make sure all icons in a button have the `sui-sm` class. That's the default size.

File: page-modals.html [hunk 2]
```diff
@@ -1749,9 +1749,9 @@ <h3 id="demo-dialog--sample-hubspot-ticket-title" class="sui-box-title sui-lg">C
 								</div>
 
 								<div class="sui-box-footer sui-flatten sui-content-right sui-spacing-top--30">
-									<button id="demo-dialog--sample-hubspot-ticket-activate" class="sui-button sui-button--solid-blue">
-										<span class="sui-loading-text">Activate</span>
-										<span class="sui-icon-loader sui-loading" aria-hidden="true"></span>
+									<button id="demo-dialog--sample-hubspot-ticket-activate" class="sui-button sui-button--solid sui-button--blue">
+										<span class="sui-button__label sui-loading-text">Activate</span>
```
Comment (iamleigh) [inline page-modals.html:1753]: @creador-dev, if I am not wrong (which in this particular case, I think I am not), the `sui-loading-text` class was deprecated in the revamp.

File: page-modals.html [hunk 3]
```diff
@@ -2318,9 +2321,9 @@ <h3 class="sui-box-title sui-lg">Welcome to Beehive Pro</h3>
 															aria-describedby="demo-dialog--sample-beehive-onboard-google-access-error"
 														/>
 
-														<button id="demo-dialog--sample-beehive-onboard-google-access-auth" class="sui-button sui-button--solid-blue">
-															<span class="sui-loading-text">Authorize</span>
-															<span class="sui-icon-loader sui-loading" aria-hidden="true"></span>
+														<button id="demo-dialog--sample-beehive-onboard-google-access-auth" class="sui-button sui-button--solid sui-button--blue">
+															<span class="sui-button__label sui-loading-text">Authorize</span>
+															<span class="sui-button__loader sui-icon-loader sui-loading" aria-hidden="true"></span>
```
Comment (iamleigh) [inline page-modals.html:2326]: @creador-dev, again, the `sui-loading-text` class was deprecated. We should remove it. Also, don't forget to include the correct/default icon size.

**Review by iamleigh — CHANGES_REQUESTED** [review]

File: assets/js/shared-ui/password.js [hunk 4]
```diff
@@ -20,7 +20,7 @@
 
 					var $button = $( this ),
 						$input  = $button.parent().find( 'input' ),
-						$icon   = $button.find( '> span' )
+						$icon   = $button.find( '> span[class^="sui-icon-"]' )
```
[the remark this answers is not in the document]
Comment (creador-dev) [reply assets/js/shared-ui/password.js:23]: @iamleigh If i use sui-button__icon then it won't work for password field as it has "sui-button-icon".

[the remark this answers is not in the document]
Comment (iamleigh) [reply assets/js/shared-ui/password.js:23]: @creador-dev, make a logical comparison to know if it is an icon button or a regular one. If it is an icon button use the regular method or else use `sui-buton__icon`.

**Review by iamleigh — CHANGES_REQUESTED** [review]

**Review by iamleigh — APPROVED** [review]

**Pull request merged**